src/localisation_igvc/launch/gps_to_odom.launch

```diff
@@ -6,10 +6,10 @@
     -->
     <node name="gps_to_odom" pkg="gps_common" type="utm_odometry_node"> 
         <param name="rot_covariance" value="99999" /> 
-        <param name="frame_id" value="base_link" /> 
+        <param name="frame_id" value="gps" /> 
  
-        <remap from="odom" to="/gps_driver/odom" /> 
-        <remap from="fix" to="/gps_driver/navsatfix" /> 
+        <remap from="odom" to="/gps/odom" /> 
+        <remap from="fix" to="/gps/navsatfix" /> 
```

Comment:
On second look, the `nmea_gps.launch` file publishes in `/gps_driver/navsatfix` no? Why are we changing this?

Reply:
Nvm, found out we changed it to this in our fork.